dnf-behave-tests/features/module-enable-errors.feature

```diff
@@ -22,6 +22,40 @@ Scenario: Fail to enable a different stream of an already enabled module
     And stderr contains "The operation would result in switching of module 'nodejs' stream '8' to stream '10'"
     And stderr contains "Error: It is not possible to switch enabled streams of a module."
 
+Scenario: Fail to install a different stream of an already enabled module
+   When I execute dnf with args "module enable nodejs:8"
+   Then the exit code is 0
+    And Transaction is following
+        | Action                   | Package            |
+        | module-stream-enable     | nodejs:8           |
+    And modules state is following
+        | Module    | State     | Stream    | Profiles  |
+        | nodejs    | enabled   | 8         |           |
+   When I execute dnf with args "module install nodejs:10"
+   Then the exit code is 1
+    And modules state is following
+        | Module    | State     | Stream    | Profiles  |
+        | nodejs    | enabled   | 8         |           |
+    And stderr contains "The operation would result in switching of module 'nodejs' stream '8' to stream '10'"
+    And stderr contains "Error: It is not possible to switch enabled streams of a module."
+
+@bz1706215
+Scenario: Fail to install a different stream of an already enabled module
```

Comment:
Please distinguish this scenario from the previous one by changing the name to something like
`Fail to install a different stream of an already enabled module using @module:stream syntax`